conda/environments/builddocs_py37.yml

```diff
@@ -4,3 +4,6 @@ channels:
 - conda-forge
 dependencies:
 - rapids-doc-env
+- distributed
+- dask
+- pandas
```

Comment:
```suggestion
- pandas
```

Reply:
We don't need `pandas` explicitly, it's a dependency of `dask`, see https://github.com/rapidsai/dask-cuda/blob/a27bd93aba8ad2d32003984ffa0c14fbf65e319f/conda/recipes/dask-cuda/meta.yaml#L26-L31 .

Reply:
An important note: `pandas` is a dependency of `dask`, but not `dask-core`, therefore we need the first one.